policyengine_us/variables/gov/states/ri/tax/income/credits/property_tax/ri_property_tax_credit.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class ri_property_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island property tax credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "http://webserver.rilin.state.ri.us/Statutes/TITLE44/44-33/44-33-9.htm"
+    )
+    defined_for = StateCode.RI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ri.tax.income.credits.property_tax
+        agi = tax_unit("adjusted_gross_income", period)
+        num_household = tax_unit("tax_unit_size", period)
+        credit = where(
+            num_household == 1,
+            p.rate.one_person.calc(agi) * agi,
+            p.rate.multiple_people.calc(agi) * agi,
+        )
+        return min_(credit, p.max_amount)
```

Comment:
we need to check whether the person has paid property taxes or rent - see new jersey property tax example